views/js/qtiCreator/widgets/interactions/inlineChoiceInteraction/Widget.js

```diff
@@ -29,12 +29,14 @@ define([
 
     InlineChoiceInteractionWidget.renderChoice = function(choice, shuffleChoice){
 
-        var tplData = {
+        const dir = choice.getRootElement().getBody().getAttributes().dir
+        let tplData = {
```

Comment:
Is tplData going to change its value? As you declare it as `let`, I will wait that its value changes.

Reply:
Assumed that the template data might change as states do, hence used let... Did I look at it the wrong way or what am I missing?

Reply:
It is an object, so you can declare it as `const` because it is not going to complain if you change things inside the object.
https://www.javascripttutorial.net/es6/javascript-const/